installer/mod.ts

```diff
@@ -140,17 +141,58 @@ async function fetchModule(url: string): Promise<any> {
 function showHelp(): void {
   console.log(`deno installer
   Install remote or local script as executables.
-  
+
 USAGE:
-  deno https://deno.land/std/installer/mod.ts EXE_NAME SCRIPT_URL [FLAGS...]  
+  deno https://deno.land/std/installer/mod.ts EXE_NAME SCRIPT_URL [FLAGS...]
 
 ARGS:
-  EXE_NAME  Name for executable     
+  EXE_NAME  Name for executable
   SCRIPT_URL  Local or remote URL of script to install
   [FLAGS...]  List of flags for script, both Deno permission and script specific flag can be used.
   `);
 }
 
+async function genereateExecutable(
+  filePath: string,
+  commands: string[]
+): Promise<void> {
+  // genereate Batch script
+  if (isWindows) {
+    const template = `
+@IF EXIST "%~dp0\deno.exe" (
+  "%~dp0\deno.exe" ${commands.slice(1).join(" ")} %*
+) ELSE (
+  @SETLOCAL
+  @SET PATHEXT=%PATHEXT:;.TS;=;%
+  ${commands.join(" ")} %*
+)
+`;
+    const cmdFile = filePath + ".cmd";
+    await writeFile(cmdFile, encoder.encode(template));
+    await chmod(cmdFile, 0o755);
```

Comment:
maybe `return` is missing after this line?

Reply:
In windows. It will generate 2 executable file. eg. `file_server` and `file_server.cmd`

`file_server` is a bash script. `file_server.cmd` is for `power shell`

Reply:
thx for review

Reply:
Thanks for clarification! makes sense!